Footer/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Commercial Project</title>
+  <link rel="stylesheet" href="Commercial Project.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous"></head>
+<body>
+  <section class="Brands">
+    <h2>Brands we distribute</h2>
+    <div class="container">
+      <div><img src="jbl.jpg" alt="jbl"></div>
+      <div><img src="LG.jpg" alt="LG"></div>
+      <div><img src="sony.jpg" alt="sony"></div>
+      <div><img src="pioneer.jpg" alt="pioneer"></div>
+      <div><img src="harman-kardon.jpg" alt="harman-kardon"></div>
+      <div><img src="Kipsch.jpg" alt="Kipsch"></div>
+    </div>
+  </section>
+  <footer>
+      <div class="footer-cont">
+        <div class="ABOUT-THE-STORE">
```

Comment:
it's better to not use uppercase with the classes name 
use one way of naming the classes